charts/apisix/values.yaml

```diff
@@ -96,6 +96,16 @@ apisix:
   podAntiAffinity:
     enabled: false
 
+  # timezone is the timezone where apisix uses.
+  # For example: "UTC" or "Asia/Shanghai"
+  # This value will be set on apisix container's environment variable TZ.
+  # You may need to set the timezone to be consistent with your local time zone,
+  # otherwise the apisix's logs may used to retrieve event maybe in wrong timezone.
+  timezone: UTC
```

Comment:
Should add the documents to explain it.

Reply:
> Should add the documents to explain it.

I also add this note on the `README` document.  I think this explanation document about `timezone` might be more appropriate in the [APISIX FAQ](https://apisix.apache.org/docs/apisix/FAQ).   @tokers  WDYT？

Reply:
Agree with you, would you like to submit another PR to add it?

Reply:
> Agree with you, would you like to submit another PR to add it?

OK，I will submit another PR first and then deal with the current PR